src/content/components/TimelineCanvas.js

```diff
@@ -112,11 +115,7 @@ export default class TimelineCanvas extends Component {
   }
 
   _onDoubleClick() {
-    const { hoveredItem } = this.state;
-    if (hoveredItem === null) {
-      return;
-    }
-    this.props.onDoubleClickItem(hoveredItem);
+    this.props.onDoubleClickItem(this.state.hoveredItem);
```

Comment:
This was a bizarre flow error. I'm not sure why I had to resort to this to be honest.